src/spv/spv_wrapper.cpp

```diff
@@ -883,6 +885,11 @@ UniValue CSpvWrapper::GetHTLCReceived(const std::string& addr)
 
     auto htlcTransactions = BRListHTLCReceived(wallet, addressFilter);
 
+    // Sort by Bitcoin address
+    std::sort(htlcTransactions.begin(), htlcTransactions.end(), [](const std::pair<BRTransaction*, size_t>& lhs, const std::pair<BRTransaction*, size_t>& rhs){
+        return strcmp(lhs.first->outputs[lhs.second].address, rhs.first->outputs[rhs.second].address) < 0;
+    });
+
```

Comment:
Why sorting? And why not in `BRListHTLCReceived` ?

Reply:
Will still need to be sorted in BRListHTLCReceived in the same way. I was thinking of adding later an option for the user to specify sort order, like TXID, time, address and so on. I could have used all three in testing but address order sufficed.